packages/container/lib/registry.js

```diff
@@ -732,31 +732,31 @@ Registry.prototype = {
   getInjections(fullName) {
     let injections = this._injections[fullName] || [];
     if (this.fallback) {
-      injections = injections.concat(this.fallback.getInjections(fullName));
+      injections.push(...this.fallback.getInjections(fullName));
```

Comment:
so you are modifying `this._injections[fullName]`, because former did not do that